### ISSUE TITLE
error with chrome extension support

### ISSUE THREAD

**Issue by aleufroy** [arqex/freezer-redux-devtools#5]
Hi,

I got the following error when using 'supportChromeExtension'

Uncaught TypeError: Cannot read property 'dispatch' of undefined.

I'm using Redux DevTools 1.0.0.17.

Looking for the code, it seems that window.devToolsExtension should be called.

**Comment by zalmoxisus** [comment]
Yes, that was deprecated several months ago, as now it is possible to send additional parameters to the extension.
